service/notification.go

```diff
@@ -0,0 +1,27 @@
+package service
+
+import "time"
+
+type (
+	Notification struct {
```

Comment:
I think there is a lot we can do differently about the schema of this object, but I think this is fine to start with for now until we have a polished event from Cadence that can be passed as a payload.

Reply:
Agreed. Created an issue to do it later https://github.com/cadence-oss/cadence-notification/issues/4